test/boost/CMakeLists.txt

```diff
@@ -207,7 +207,8 @@ add_scylla_test(rust_test
   KIND BOOST
   LIBRARIES inc)
 add_scylla_test(s3_test
-  KIND SEASTAR)
+  KIND SEASTAR
+  LIBRARIES scylla_encryption)
 add_scylla_test(gcp_object_storage_test
```

Comment:
gcp_object_storage_test should need the scylla_encryption as well. no?

Reply:
you are right
fixed